pdr_backend/statutil/dash_plots/view_elements.py

```diff
@@ -32,31 +49,36 @@ def get_column_graphs(figures: list[dict], title: str, tooltip: str):
                 ),
                 dbc.Tooltip(
                     dcc.Markdown(tooltip),
-                    target=figures[0]["graph_id"] + "-title",
+                    target=id_parent + "-title",
                 ),
-                *[
-                    dcc.Graph(
-                        figure=fig["fig"],
-                        id=fig["graph_id"],
-                        style={
-                            "width": "100%",
-                            "height": str(fig.get("height", height_percentage)) + "vh",
-                            "padding": "0",
-                        },
-                    )
-                    for fig in figures
-                ],
-            ],
+            ]
+            + figures,
             style={
                 "display": "flex",
                 "flexDirection": "column",
                 "alignItems": "center",
                 "justifyContent": "center",
+                "width": "100%",
             },
         ),
     )
 
 
+def get_table(columns, data):
+    return dash_table.DataTable(
```

Comment:
@KatunaNorbert I suggest you also take a look at the LakeInfo dashboard. I used the dbc table component and I liked it more than the DataTable from Dash. We can of course have different elements, but maybe you'll like that one better too :)

Reply:
I looked at that one but turns out that it doesn't support selection out of the box